ui/mods/mod_reforged/reforged_assets.js

```diff
@@ -6,3 +6,11 @@ Reforged.Asset = {
 	BUTTON_RETREAT_WIN : 'ui/skin/icon_rf_retreat_win.png',
 	MAIN_MENU_LOGO: 'ui/skin/rf_main_menu_logo.png'
 };
+
+ProgressbarStyleIdentifier["rf_Reach"] = 'rf_reach';
+ProgressbarValueIdentifier["rf_Reach"] = 'rf_reach';
+ProgressbarValueIdentifier["rf_ReachMax"] = 'rf_reachMax';
+ProgressbarValueIdentifier["rf_ReachLabel"] = 'rf_reachLabel';
```

Comment:
These 4 lines might not be needed anymore if that progressbar call further up was from an early prototype.

Reply:
I was fixated on the combat overlay only and couldn't make sense of this change. But it actually belongs to the character screen